sympy/core/tests/test_relational.py

```diff
@@ -802,11 +802,15 @@ def test_simplify_relational():
 
     # Test particular branches of _eval_simplify
     m = exp(1) - exp_polar(1)
-    assert simplify(m*x > 1) is S.false
     # These two test the same branch
     assert simplify(m*x + 2*m*y > 1) is S.false
     assert simplify(m*x + y > 1 + y) is S.false
```

Comment:
It would be better to change these tests as described in https://github.com/sympy/sympy/pull/22111#discussion_r711581320.

These are bugs on master:
```python
In [25]: class A(Expr):
    ...:     is_commutative = True
    ...: 

In [26]: simplify(A()*x > 1)
Out[26]: False

In [27]: simplify(A()*x + 2*A()*y > 1)
Out[27]: False
```
This PR fixes the first bug but not the second:
```python
In [2]: simplify(A()*x > 1)
Out[2]: x⋅A() > 1

In [3]: simplify(A()*x + 2*A()*y > 1)
Out[3]: False
```